pkg/adaptor/hypervisor/libvirt/libvirt.go

```diff
@@ -178,7 +178,7 @@ func CreateInstance(c context.Context, libvirtClient *libvirtClient, v *vmConfig
 		var addr string
 		addrTail := v.num
 		switch {
-		case addrTail >= 2 && addrTail < 10:
+		case addrTail >= 0 && addrTail < 10:
```

Comment:
I think libvirt will automatically generate one with 52:54: prefix if one is not specified in the guest XML. Is it ok to remove the generation logic altogether ? I thought it could be helpful to follow a standard format for mac address for pod vms for debugging, but it can also be determined by just looking at the qemu cmd line.

Reply:
I think it's perfectly fine to remove the MAC generation logic.

Reply:
Please take a look at this version, I removed the logic altogether. Tested with a F36 host for the pod vm.